### DISCUSSION TITLE
appearance-button

### DISCUSSION THREAD

**Discussion by darethas** [tailwindlabs/tailwindcss#9109] [Ideas]
A common pattern to create a button that acts as a link is to simply style the anchor tag as a button by using `apperance: button`.

I would like to suggest a tailwind utility called `appearance-button`, which would also fit along with the existing `appearance-none` utility.

Using a button example from tailwind-ui but as an anchor tag instead, you could use it as so:

```html
<a href="/foo" class="apperance-button inline-flex items-center justify-center rounded-md border border-transparent bg-indigo-800 px-4 py-2 text-sm font-medium text-white shadow-sm hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-red-500 focus:ring-offset-2 sm:w-auto">New Post</a>
```    
For a nicely styled button that is also a link.

**Comment by kimhout-sam** [comment]
yeah i also need `appearance-textfield` for remove native stepUp/stepDown and using custom one
```tsx
{/* react code */}
<input type="number" className="appearance-textfield" />
{/* react code */}
```
